modules/nw-ingress-setting-a-custom-default-certificate.adoc

```diff
@@ -13,7 +13,7 @@ custom resource (CR).
 
 * You must have a certificate/key pair in PEM-encoded files, where the
 certificate is signed by a trusted certificate authority or by a private trusted
-certificate authority that you have configured.
+certificate authority you configured in a custom PKI.
```

Comment:
@codyhoag I believe when we discussed writing for translation at one point we suggested using words such as `that` in place: 
`certificate authority that you configured in a custom PKI.`

Reply:
@mburke5678 ah, got it! That makes sense; I'll add it back.